wiki/markdown/query.md

```diff
@@ -36,39 +36,39 @@ Examples of this endpoint using curl
 
 Examples of criterias:
 
-    // This returns all objects where the object.testCaseExecutions.outcome.id is"TC5"
```

Comment:
The paragraph with the section header ## Perform freestyle query on created aggregated object needs change also. The first sentence there is: "It is possible to query for documents using freestyle queries." it should be: "It is possible to queries on aggregated objects using freestyle queries."

Reply:
after "to" first form or "ing" form.

Anyway, I have changed it to "make queries"

Reply:
Yes my mistake, looks better :)